internal/graph/graph_test.go

```diff
@@ -708,6 +708,74 @@ func TestConnectedObjectGraph_RelationshipIngresss(t *testing.T) {
 				},
 			},
 		},
```

Comment:
Would you mind adding a test that involves different typed wildcards within a union rewrite definition? For example

```
type user
type employee
type group

type document
    relations
      define relation1: [user:*] or relation2 or relation3 or relation4
      define relation2: [group:*]
      define relation3: [employee:*]
      define relation4: [user]
```
expectation
```
RelationshipIngresses(document#relation1, user) --> [
  document#relation1 (direct),
  document#relation4 (direct)
]
```

Another decent one:
```
type user

type document
    relations
      define relation1: [user] or relation2
      define relation2: [user:*]
```
expectation
```
RelationshipIngresses(document#relation1, user:*) --> [
  document#relation2 (direct)
]
```

Reply:
Tests added. Thanks for that!